src/components/TodoContext/TodoContext.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { getTodos, getUser } from '../../api';
+import { User } from '../../types/User';
+import { Context } from '../../types/Context';
+import { Todo } from '../../types/Todo';
+import { Status } from '../../types/Status';
+import { filterData } from '../services';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.